python/lsst/faro/measurement/TractMeasurementTasks.py

```diff
@@ -27,29 +52,30 @@ class WPerpTask(Task):
     _DefaultName = "WPerpTask"
 
     def run(
-        self, metricName, catalogs, photoCalibs=None, astromCalibs=None, dataIds=None
+        self, metricName: str, data: Dict[str, List[CalibratedCatalog]],
     ):
         self.log.info("Measuring %s", metricName)
-        bands = set([f["band"] for f in dataIds])
+        bands = set('gri')
```

Comment:
This file seems to be using `"` for all the string quoting so can you use `"` here as well please?